react/UploadQueue/styles.styl

```diff
@@ -0,0 +1,161 @@
+@require 'settings/palette.styl'
+@require 'components/popover.styl'
+@require 'settings/icons.styl'
+@require 'settings/breakpoints.styl'
+@require 'settings/z-index.styl'
+
+$coz-bar-size=3rem
+
+.upload-queue
+    border   .0625rem solid var(--silver)
+    border-radius    .25rem
+    background-color var(--white)
+    display          flex
+    flex-direction   column
+    transition       0.5s
+    opacity          0
+    visibility       hidden
+    transform        translateY(18.75rem)
+
+.upload-queue--popover
+    @extend $popover
+    position fixed
+    bottom   .5rem
+    right    1.5rem
+    height   13.125rem
+    width    30rem
+    max-width 90%
+
+.upload-queue--visible
+    opacity    1
+    visibility visible
+    transform  translate(0)
+
+.upload-queue-header
+    background-color var(--paleGrey)
+    color            var(--charcoalGrey)
+    font-weight      bold
+    margin           0
+    padding          .5rem 1rem .5rem
+
+    .upload-queue-header-inner
+        display flex
+        justify-content space-between
+        align-items center
+
+    .btn-close
+        border 0
+        background 0
+        margin 0
+        padding .4rem 0 .4rem .8rem
+        color var(--dodgerBlue)
+        text-transform uppercase
+        font-size .8rem
+
+progress.upload-queue-progress
+    -webkit-appearance none
+    -moz-appearance    none
+    border     none
+    display    block
+    width      100%
+    height     .125rem
+    margin     0
+
+progress.upload-queue-progress::-webkit-progress-bar
+    background-color var(--silver)
+progress.upload-queue-progress::-webkit-progress-value
+    background-color var(--dodgerBlue)
+progress.upload-queue-progress::-moz-progress-bar
+    background-color var(--dodgerBlue)
```

Comment:
Do we still need this style since we're using Tresholdbar?

Reply:
Yes, because they are the styles for the overall progress bar.